packages/docusaurus-migrate/src/types.ts

```diff
@@ -35,6 +35,7 @@ export interface VersionTwoConfig {
   url: string;
   organizationName?: string;
   projectName?: string;
+  noIndex?: string;
```

Comment:
should be a boolean, the existing v1 config type looks bad too, both should be booleans (not not change anything at runtime anyway)

Reply:
No problem